ArduCopter/tuning.cpp

```diff
@@ -26,9 +24,9 @@ void Copter::tuning()
         return;
     }
 
-    const uint16_t radio_in = rc_tuning->get_radio_in();
-    float tuning_value = linear_interpolate(g2.tuning_min, g2.tuning_max, radio_in, rc_tuning->get_radio_min(), rc_tuning->get_radio_max());
-    
+    const float control_in = rc_tuning->get_control_in_zero_dz();
```

Comment:
```suggestion
    const float control_in = rc_tuning->get_control_in_zero_dz();
```
norm_input_ignore_trim